### PR TITLE
Docs/ (Procedures) approve transfer erc20 checkpoint

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

File: src/procedures/LaunchTieredSto.ts [hunk 1]
```diff
@@ -20,9 +20,31 @@ import { findEvents, isValidAddress } from '../utils';
...
 import { ZERO_ADDRESS } from '../utils/constants';
 
+/**
+ * Procedure to launch a tiered STO module with necessary initialization data, transferring poly setup cost to the security token before doing so.
+ * As part of the procedure, preissuing can be enabled in 3.1.0 version Security Token
+ */
 export class LaunchTieredSto extends Procedure<LaunchTieredStoProcedureArgs, TieredSto> {
   public type = ProcedureType.LaunchTieredSto;
 
+  /**
```
Comment (monitz87) [inline src/procedures/LaunchTieredSto.ts]:
```
- Transfer the necessary amount of POLY to the Security Token to cover the STO's setup fee
- Launch the Tiered STO
- Allow pre-issuing (if applicable, defaults to false)
- Return the newly created STO

Note that this procedure will fail if:
- The supplied custom currency oracle addresses corresponding to the selected fund raise currencies are invalid
- Raising in Stable Coin and not providing stable coin addresses
- Attempting to allow pre-issuing on a version 3.0 STO
```

File: src/procedures/SignDisableControllerAck.ts [hunk 2]
```diff
@@ -2,9 +2,15 @@ import { Procedure } from './Procedure';
 import { ProcedureType, ErrorCode, SignDisableControllerAckProcedureArgs } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure to sign an acknowledgement for disabling the controller of a security token
```
Comment (monitz87) [inline src/procedures/SignDisableControllerAck.ts]: Procedure that signs an acknowledgement to permanently disable a Security Token's Controller functionality

File: src/procedures/SignDisableControllerAck.ts [hunk 3]
```diff
@@ -2,9 +2,15 @@ import { Procedure } from './Procedure';
...
+ */
 export class SignDisableControllerAck extends Procedure<SignDisableControllerAckProcedureArgs> {
   public type = ProcedureType.SignDisableControllerAck;
 
+  /**
+   * - Sign an acknowledgement to disable the controller of the security token
```
Comment (monitz87) [inline src/procedures/SignDisableControllerAck.ts]: Sign data to confirm the intent of permanently disabling the Security Token's Controller functionality

(delete hyphen)

File: src/procedures/SignFreezeIssuanceAck.ts [hunk 4]
```diff
@@ -2,9 +2,15 @@ import { Procedure } from './Procedure';
 import { ProcedureType, ErrorCode, SignFreezeIssuanceAckProcedureArgs } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure to sign an acknowledgement for freezing the issuance of a security token
```
Comment (monitz87) [inline src/procedures/SignFreezeIssuanceAck.ts]: Procedure that signs an acknowledgement to permanently freeze issuance of a Security Token

File: src/procedures/SignFreezeIssuanceAck.ts [hunk 5]
```diff
@@ -2,9 +2,15 @@ import { Procedure } from './Procedure';
...
+ */
 export class SignFreezeIssuanceAck extends Procedure<SignFreezeIssuanceAckProcedureArgs> {
   public type = ProcedureType.SignFreezeIssuanceAck;
 
+  /**
+   * - Sign an acknowledgement to freeze the issuance of the security token
```
Comment (monitz87) [inline src/procedures/SignFreezeIssuanceAck.ts]: Sign data to confirm the intent of permanently disabling issuance of the Security Token

(delete hyphen)

File: src/procedures/TransferErc20.ts [hunk 6]
```diff
@@ -24,6 +27,18 @@ export const createTransferErc20Resolver = (
 export class TransferErc20 extends Procedure<TransferErc20ProcedureArgs> {
   public type = ProcedureType.TransferErc20;
 
+  /**
```
Comment (monitz87) [inline src/procedures/TransferErc20.ts]:
```
Transfer an ERC20 token to another wallet. The token in question defaults to POLY if no address is supplied

Note that the procedure will fail if:
- The owner's token balance is less than the amount being transferred. The only exception to this is when transferring POLY on a testnet. If that is the case, an extra transaction will be submitted to request the missing amount of tokens from the faucet
- The token being transferred is a Security Token. In that case, the corresponding Security Token transfer procedures should be used
```

File: src/procedures/CreateDividendDistribution.ts [hunk 7]
```diff
@@ -17,8 +17,8 @@ import { findEvents } from '../utils';
 import { SecurityToken, DividendDistribution } from '../entities';
 
 /**
- * Procedure to create a dividend distribution on a security token with an enabled ERC20 Dividends Checkpoint module
- * Optionally setting tax with holdings for investors represented with percentage
+ * Procedure to create a Dividend Distribution on a Security Token.
+ * The funds to be distributed as dividends will come from the current user's wallet Optionally setting tax with holdings for investors represented with percentage
```
Comment (monitz87) [inline src/procedures/CreateDividendDistribution.ts]: Delete everything from "Optionally" onwards

File: src/procedures/CreateDividendDistribution.ts [hunk 8]
```diff
@@ -27,15 +27,15 @@ export class CreateDividendDistribution extends Procedure<
   public type = ProcedureType.CreateDividendDistribution;
 
   /**
-   * - Approve the ERC20 Dividends Checkpoint module to spend the amount equal to the dividend amount in the procedure arguments
+   * - Approve spend of the amount that will be distributed
```
Comment (monitz87) [inline src/procedures/CreateDividendDistribution.ts]: Delete emtpy lines between items (do this accross the whole PR)

**Comment by victorwiebe** [comment]
Should be okay now

**Pull request merged**